### DISCUSSION TITLE
Mobile support?

### DISCUSSION THREAD

**Discussion by alexmozaidze** [janet-lang/janet#1323] [Q&A]
I saw that Janet has bindings to Raylib, and I am curious if I can compile Janet to mobile platforms at all. Is it possible to compile for Android, or is it not yet supported?

**Comment by sogaiu** [comment]
I haven't seen anything along these lines recently (assuming you mean building a whole app).  Possibly @ahungry might have done something at some point.

FWIW, there is [support for termux](https://github.com/janet-lang/janet/pull/840), but I guess this is not the sort of thing you had in mind.

Would be happy to learn if someone has gotten something working on the app end of things.

**Reply by sogaiu** [reply]
So [here](https://github.com/ahungry/janet-android) is one of the proof-of-concept attempts by ahungry.  IIUC, it's written in Kotlin but calls an embedded Janet interpreter.
